### ISSUE TITLE
gzipper.js:6 Uncaught TypeError: Cannot read property 'expose' of undefined

### ISSUE THREAD

**Issue by ivictbor** [andywer/threads.js#127]
Thx for threads!

"threads": "^1.0.0-beta",

React + react-app-rewired + webpack threads-plugin

```
const threads = require('threads/worker');
//threads is undefined 
```

```
const threads = require('threads');
// threads correct and has expose function 
```

Probably need to update doc?

**Comment by andywer** [comment]
Ohh, no. The docs are fine. The `worker.js` file is not, though.

Thanks for pointing out! Will fix it right away.

**Comment by andywer** [comment]
Published a new version containing the fix: `threads@1.0.0-beta.1`

**Comment by ivictbor** [comment]
@andywer confirm that it works now, thank you for a fast fix! Great job
